specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -526,4 +526,63 @@ interface DevBoxes {
   @doc("Gets an operation on a Dev Box.")
   @added(APIVersions.v2024_02_01)
   getOperation is StandardResourceOperations.ResourceRead<DevBoxOperation>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "No model for this."
+  @doc("Gets the log for an imaging customization task.")
+  @route("/projects/{projectName}/catalogs/{catalogName}/imageDefinitions/{imageDefinitionName}/builds/{buildName}/customizationGroups/{customizationGroupName}/logs/{customizationTaskId}")
+  @get
+  @added(APIVersions.v2024_08_01_preview)
+  getImagingCustomizationTaskLog is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+
+      @doc("The name of the catalog hosting the image definition.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      catalogName: string;
+
+      @doc("The name of the image definition.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      imageDefinitionName: string;
+
+      @doc("The name of the build.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      buildName: string;
+
+      @doc("A customization group name.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      customizationGroupName: string;
+
+      @doc("A customization task ID.")
+      @maxLength(36)
+      @minLength(36)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$")
+      @path
+      customizationTaskId: string;
+    },
+    {
+      @doc("Set producer to text/plain")
+      @header
+      contentType: "text/plain";
```

Comment:
Hey all, so we've just noticed that adding this here results in this being generated in the swagger:

```
"produces": [
    "text/plain",
    "application/json"
],
```

The issue is we only want `text/plain` and not `application/json`. This is what we've had in our old hand-written swagger. The `application/json` I believe is coming from the default error response from the TypeSpec. However, this is casuing issues when we generate the SDK, it will generate this:

```
using var message = CreateGetCustomizationTaskLogRequest(endpoint, projectName, userId, devBoxName, customizationGroupName, customizationTaskId);
    await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
    switch (message.Response.Status)
    {
        case 200:
            {
                StreamReader streamReader = new StreamReader(message.Response.ContentStream);
                string value = await streamReader.ReadToEndAsync().ConfigureAwait(false);
                return Response.FromValue(value, message.Response);
            }
        case 200:
            {
                string value = default;
                using var document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
                value = document.RootElement.GetString();
                return Response.FromValue(value, message.Response);
            }
        default:
            throw await _clientDiagnostics.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);
    }

```

The two 200 cases cause a compilation error. Would appreciate any help in modifying the TypeSpec so that this doesn't happen.

Reply:
What is the content type for a 4xx response? Presumably JSON. Which means that the `produces` is correct.